universal-login-web3/lib/index.ts

```diff
@@ -32,6 +33,7 @@ const WALLET = {
 export class ULWeb3Provider implements Provider {
```

Comment:
It is time to get rid of `WALLET` and use the one from WalletService.

Reply:
Yes, it's in the next PR